network-api/networkapi/wagtailpages/tests/buyersguide/test_utils.py

```diff
@@ -0,0 +1,46 @@
+from networkapi.wagtailpages.factory import buyersguide as buyersguide_factories
+from networkapi.wagtailpages.pagemodels.buyersguide.utils import get_bg_featured_cta
+from networkapi.wagtailpages.tests import base as test_base
+
+
+class TestGetFeaturedCallToActionFunction(test_base.WagtailpagesTestCase):
```

Comment:
Nice call. Thanks for adding tests for this function directly! I think that is a great idea 👍